js/renovation/component_wrapper/check_box.ts

```diff
@@ -27,4 +27,9 @@ export default class CheckBox extends Editor {
 
     this._invalidate();
   }
+
+  setAria(name: string, value: string): void {
+    const attrName = this._getAriaName(name);
+    (this.$element() as any).attr(attrName, value);
```

Comment:
May be (this.$element() as any) -> (this.$element() as HTMLDivElement)

Reply:
$element() returns our wrapper, not native HTMLDivElement.
This wrapper implements JQuery interface.

It's very strange but lint warns about non [consistent type](https://github.com/DevExpress/DevExtreme/pull/16642/checks?check_run_id=2094488713)
![image](https://user-images.githubusercontent.com/57402891/110954780-f32be400-8359-11eb-940a-50df57fc4809.png)

Reply:
Fixed via using of the addAttributes method in our utils..ts